charters/charter-2021.html

```diff
@@ -0,0 +1,841 @@
+            <dd>The Web Real-Time Communications Working Group develops APIs for media capture, media recording and media processing.</dd>
+
+<!-- CGs -->
+            <dt><a href="https://www.w3.org/community/colorweb/">Color on the Web CG</a></dt>
+            <dd>The Color on the Web Community Group discussed support for High-Dynamic Range, wide-gamut color and related topics on the web.</dd>
+
+            <dt><a href="https://www.w3.org/community/games/">Games CG</a></dt>
+            <dd>The Games Community Group tracks technologies needed for the development of games on the Web.</dd>
+
+            <dt><a href="https://www.w3.org/community/httpslocal/">HTTPS in Local Network CG</a></dt>
+            <dd>The HTTPS in Local Network Community Group (CG)
+            explores the manner of secure communication between
+            browsers and server-capable devices in local network such
+            as set-top boxes and network attached storages.</dd>
+
+            <dt><a href="https://www.w3.org/community/immersive-web/">Immersive Web CG</a></dt>
+            <dd>The Immersive Web Community Group incubates additional proposals to interact with XR devices and sensors in browsers.</dd>
+            
+            <dt><a href="https://www.w3.org/community/inbandtracks/">Media Resource in-band Tracks CG</a></dt>
+            <dd>How user agents should expose in-band tracks as HTML5 media element video, audio and text tracks so that Web applications can access the in-band track information</dd>
+
+            <dt><a href="https://www.w3.org/community/webtiming/">Multi-device Timing CG</a></dt>
+            <dd>Support multi-device timing in media applications, such as multi-screen presentations</dd>
+
+
+
+            <dt><a href="https://www.w3.org/community/webscreens/">Second Screen CG</a></dt>
+            <dd>The Second Screen Community Group, companion group of the Second Screen Working Group, discusses the development of an Open Screen Protocol to improve the interoperability between first and second screens.</dd>
+
+            <dt><a href="https://www.w3.org/community/wicg/">Web Platform Incubator CG</a></dt>
+            <dd>The Web Platform Incubator Community Group incubates a number of proposals from the community, including media-relevant ideas such as <a href="https://github.com/WICG/canvas-color-space">Canvas Color Spaces</a>, <a href="https://wicg.github.io/video-rvfc/">HTMLMediaElement.requestVideoFrameCallback()</a>, and <a href="https://wicg.github.io/media-feeds/">Media Feeds</a>.</dd>
+
+            <dt><a href="https://www.w3.org/community/webmediaapi/">Web Media API CG</a></dt>
+            <dd>The Web Media API Community Group incubates a minimum set of web technologies that developers of media web applications can rely on being supported across devices. The group was proposed by and continues to cooperate with the <a href="https://standards.cta.tech/kwspub/wave/">CTA WAVE Project</a>.</dd>
```

Comment:
Cool URLs don't change, but unfortunately's CTA WAVE's URL has:
```suggestion
            <dd>The Web Media API Community Group incubates a minimum set of web technologies that developers of media web applications can rely on being supported across devices. The group was proposed by and continues to cooperate with the <a href="https://cta.tech/WAVE">CTA WAVE Project</a>.</dd>
```